herddb-core/src/main/java/herddb/model/Transaction.java

```diff
@@ -128,6 +128,9 @@ public LockHandle lookupLock(String tableName, Bytes key) {
     }
 
     public void registerLockOnTable(String tableName, LockHandle handle) {
+        if (handle == null) {
```

Comment:
this is not good

Reply:
Needed to let LockHandle in NullLockManager to be null

Reply:
This can lead to errors in core execution paths.
It is better to have a dummy LockHandle, only one singleton instance